src/Lachain.Core/Blockchain/VM/ExternalHandler.cs

```diff
@@ -74,13 +78,18 @@ private static ulong GetDeployHeightV2(ulong currentHeight, UInt160 contract, UI
                     input, gasLimit, message).ReturnValue;
                 Logger.LogInformation($"GetDeployHeight result :[{(height != null ? height.ToHex() : "null")}]");
                 if (HardforkHeights.IsHardfork_6Active(currentHeight))
-                    height = height ?? new byte[64];
+                {
+                    height ??= HardforkHeights.IsHardfork_8Active(currentHeight) ? HardforkHeights.GetHardfork_3().ToBytes().ToArray() : new byte[64]; 
+                    if(height.Length < 1)
```

Comment:
I think there is a problem here. If height.Length == 0 then it is an empty array which is not equivalent to null I suppose. So in the previous implementation it will remain the same. But now we are assigning new byte[64] if hardfork8 is not active.

Reply:
I think it is better to put a single if ( HardforkHeights.IsHardfork_8Active(currentHeight) ) and handle everything there.